core/cmd/remote_client_test.go

```diff
@@ -1430,3 +1433,37 @@ func TestClient_RunOCRJob_JobNotFound(t *testing.T) {
 	require.NoError(t, client.RemoteLogin(c))
 	assert.EqualError(t, client.TriggerPipelineRun(c), "500 Internal Server Error; no job found with id 1 (most likely it was deleted)")
 }
+
+func TestClient_ListJobsV2(t *testing.T) {
+	t.Parallel()
+
+	app, cleanup := cltest.NewApplication(t)
```

Comment:
I'm just `NewApplication` here instead of the set up I've seen in other tests (seen below)

```
	rpcClient, gethClient, _, assertMocksCalled := cltest.NewEthMocksWithStartupAssertions(t)
	defer assertMocksCalled()
	config, cleanup := cltest.NewConfig(t)
	defer cleanup()
	app, cleanup := cltest.NewApplicationWithConfig(t, config,
		eth.NewClientWith(rpcClient, gethClient),
	)
	defer cleanup()
	require.NoError(t, app.Start())
```

Is there a reason to use `NewEthMocksWithStartupAssertions`?

Reply:
The main reason is if you want to mock some specific responses to the eth client's methods. In these tests you don't need to do that. The reason that the tests currently have it is because we did a big migration to use mockery everywhere instead of handrolled mock objects and essentially the search and replace was easier to just default to the lower level version you see there

Reply:
Actually might be some good cleanup to just use cltest.NewApplication(t) instead where we can in this test file